frontend/resourceadm/pages/DeployResourcePage/DeployResourcePage.tsx

```diff
@@ -189,22 +156,22 @@ export const DeployResourcePage = ({ navigateToPageWithError }: Props) => {
    *
    * @returns a boolean for if it is possible
    */
-  const isDeployPossible = (type: 'test' | 'prod'): boolean => {
+  const isDeployPossible = (type: 'test' | 'prod', envVersion: string): boolean => {
     if (
       type === 'test' &&
-      !hasResourceError &&
+      validateResourceData.status === 200 &&
```

Comment:
As mentioned above, the status checks is on the actual object returned from backend. It can be 400 even if it is in the success (200) part of the API Call.